src/components/ChatLog.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import ChatEntry from './ChatEntry'
+
+function ChatLog(props) {
+    const data = props.entries;  
+    return (
+    <>
+    {data.map((entries)=>{
+
+    return (
+    
+    <ChatEntry
```

Comment:
So you see how setting all of those props took up a lot of lines of code? We could make that more concise with this here: 
```javascript
 < ChatEntry toggleLike={props.toggleLike} key={entry.id} entry={entry} />
```
Then in each `chatEntry` we could access any prop we needed with `props.entry.myProp`